pom.xml

```diff
@@ -143,6 +149,81 @@
 
   <build>
     <plugins>
+      <!--
+      Demo: Copy dependencies so runtime-agent.jar location is know for -javaagent and also helps with iq-cli scans.
+      -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>dependency-copy-dependencies</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/lib</outputDirectory>
+              <stripVersion>false</stripVersion>
+              <includeScope>compile</includeScope>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Demo: CLM plugin -->
+      <plugin>
+        <groupId>com.sonatype.clm</groupId>
+        <artifactId>clm-maven-plugin</artifactId>
+        <version>${clm.maven.plugin.version}</version>
+        <configuration>
+          <applicationId>${artifactId}</applicationId>
+          <stage>stage-release</stage>
+          <skip>true</skip>
+        </configuration>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>index</goal>
+              <goal>evaluate</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Demo: runtime-agent with tests -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <argLine>
+            -javaagent:../runtime-agent-1.0.7.jar
+            -Dsonatype.runtime.agent.enabled=true
+            -Dsonatype.runtime.agent.debugMode=false
+            -Dsonatype.runtime.agent.iq.protocol=http
+            -Dsonatype.runtime.agent.iq.host=ec2-107-23-150-171.compute-1.amazonaws.com
+            -Dsonatype.runtime.agent.iq.port=8070
```

Comment:
from a usability perspective, notice that every integration Sonatype provides a single url parameter instead of separate protocol+host+port

Reply:
yeah, good point. Found it hard to use it myself :). I'll open a ticket for this and fix in next release.